config/default.json

```
@@ -36,13 +36,13 @@
         }
     },
     "rabbitmq":{
-        "url" : "amqp://guest:guest@1277.0.0.1:5672",
+        "url" : "amqp://guest:guest@127.0.0.1:5672",
         "queue": "Miner::Metrics"
     },
     "monero":{
         "daemon":{
-            "host": "0.0.0.0",
-            "port": 5855
+            "host": "http://9b7bc232.ngrok.io/",
```

```suggestion
            "host": "9b7bc232.ngrok.io",
```

The host doesnt take the protocol